tgui/packages/tgui/interfaces/NtosNetChat.jsx

```diff
@@ -175,8 +176,10 @@ export const NtosNetChat = (props) => {
                   backgroundColor={this_client && this_client.muted && 'red'}
                   height="22px"
                   placeholder={
-                    (this_client && this_client.muted && 'You are muted!') ||
-                    'Message ' + title
+                    (this_client &&
+                      this_client.muted &&
+                      'Ты в муте, дурачек!') ||
+                    'Написать в ' + title
```

Comment:
```suggestion
                    (this_client && this_client.muted && 'Вы в муте!') ||
                    'Написать в ' + title
```
Жертвуем смешнявкой ради преттиер